Demo/Demo/Networking/MerchantIntegration.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-enum MerchantIntegration {
+enum MerchantIntegration: String, CaseIterable {
```

Comment:
Why do we need case iterable?

Reply:
I iterate through them for Picker in FeatureSelectionView